assets/css/events.css

```diff
@@ -1,118 +1,113 @@
-        margin-bottom: 0;
-    }
-
-
-.upcoming a.event {
-    background-color: #ebb9ff !important;
-    box-shadow: inset 0 0 0 1px #c89dd9 !important;
-}
-
-    a.event {
-        display: block;
-        color: #000;
-        background-color: #f5f5f5;
-        height: 140px;
-        box-shadow: inset 0 0 0 1px #e1e1e1;
-        width: 100%;
-        border-radius: 3px;
-    }
-
-        a.event img {
-            float: left;
-            width: 120px;
-            padding: 10px;
-        }
-
-        a.event .description {
-            display: block;
-            position: relative;
-            height: 104px;
-            padding: 16px 10px 20px 10px;
-            white-space: nowrap;
-            text-align: left;
-        }
-
-            a.event .description h2 {
-                margin: 0 0 15px 0;
-                height: 30px;
-                line-height: 30px;
-                font-size: 24px;
-                overflow: hidden;
-                text-overflow: ellipsis;
-                font-weight: 400;
-            }
-
-            a.event .description .location {
-                margin: 15px 0;
-                height: 22px;
-                line-height: 22px;
-                font-size: 18px;
-                overflow: hidden;
-                text-overflow: ellipsis;
-            }
-
-            a.event .description .date {
-                margin: 15px 0 0 0;
-                height: 22px;
-                line-height: 22px;
-                font-size: 18px;
-                overflow: hidden;
-                text-overflow: ellipsis;
-            }
-
-
-/* Mobile */
-@media screen and (max-width: 736px) {
-    #events ul {
-        padding: 30px 16px;
-    }
-
-
-    a.event {
-        height: 80px;
-    }
-
-        a.event img {
-            width: 68px;
-            padding: 6px;
-        }
-
-        a.event .description {
-            height: 60px;
-            padding: 4px 10px 20px 5px;
-        }
-
-            a.event .description h2 {
-                margin: 0 0 2px 0;
-                line-height: 22px;
-                height: 22px;
-                font-size: 18px;
-            }
-
-            a.event .description .location {
-                margin: 2px 0;
-                font-size: 15px;
-            }
-
-            a.event .description .date {
-                margin: 2px 0 0 0;
-                font-size: 15px;
-            }
-}
\ No newline at end of file
+    padding: 10px;
+    font-size: 12px;
+}
+
+.event-facebook {
+    margin: 0 5px 0 0;
+    width: 21px;
+    height: 21px;
+}
+
+.event-facebook-wrap {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+
+.event-list {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+    padding: 0 15px 0 15px;
+}
+
+.event {
+    width: 300px;
+    height: 300px;
+    margin: 15px;
+    /*box-shadow: 0 1px 2px 0 rgba(0,0,0,.15);*/
+    box-shadow: 0 1px 0 0 rgba(0,0,0,.1);
```

Comment:
You could maybe tweak a little with the shadows as the events' details description part is the same color as the background and I think it becomes exhausting for the eye to follow.